multiples.js

```diff
@@ -7,6 +7,11 @@ exports.sumOfAMultiple = function( n ) {
   let sum = 0;
 
   // do your work here
-  
+  for (let i=0; i<n; i++){
+    if((i%3===0) || (i%5===0) || (i%3===0 && i%5===0)){
```

Comment:
For this particular exercise, you just need to check 3 and 5. You don't need to check 3 and 5.